README.md

```diff
@@ -1,10 +1,29 @@
 # sn-treemap
 Treemap Supernova
 
+## Mobile
+### Preamble
+Mobile uses react-native.   As such all javascript should work on both the web and mobile. However react-native uses a custom JavaScript engine called Hermes.  For rendering picasso charts, mobile uses a hightly optimized rendering engine, called [Helium](https://github.com/qlik-oss/react-native-helium), that uses a GPU backend for Skia.  Skia is the same 2D graphics library used for Canvas in Chrome and Android.  So again, things must be tested to ensure compatiblity.
+
+### Testing mobile
+#### Pre-reqs
+1. Install [react-native dev environment](https://reactnative.dev/docs/0.68/environment-setup). This includes Xcode and Android Studio.
+2. A QCS Account with an API Key
```

Comment:
I am a bit unsure what this means (any link with more information)
(I have not tried to look for more info myself yet, maybe it is easy to find if you search for it)

Reply:
The link to setup the react native dev env is provided in step 1.
How to setup an API Key is here https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Admin/mc-generate-api-keys.htm